code-injection-analysis/UOS2021_U.O.S-Server/uos.js

```
@@ -0,0 +1,2481 @@
+                }
+                else{
+
+                    console.log('주문 내역 성공');
+
+
+                    var response_obj = new Object();
+                    response_obj.response_code = "0012";
+
+                    var order_list_array = new Array();
+                    for(var i =0; i < result.length; i++){
+                        var obj = new Object();
```

Biome (2.1.2)

[error] Shouldn't redeclare 'select_query'. Consider to delete it or rename it.

'select_query' is defined here:


(lint/suspicious/noRedeclare)